### ISSUE TITLE
Change current directory to workspace

### ISSUE THREAD

**Issue by hseeberger** [scala-ide/scala-worksheet#102]
The current directory of the worksheet is not the workspace, but the installation directory of Eclipse:

  val current = new File(".")                     //> current  : java.io.File = .
  current.getCanonicalPath                        //> res0: String = /Users/heiko/tools/eclipse/eclipse-3.7.2-2.10.0-RC1/Eclipse.app/Contents/MacOS

**Comment by metasim** [comment]
I'm interested to know if this is still the only way of working around this problem:

http://stackoverflow.com/a/21812892/296509

**Comment by sschaef** [comment]
yes, there is no way to configure working directory from outside the worksheet.
